src/cache.rs

```diff
@@ -0,0 +1,25 @@
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct JustfileCache {
+  /// Only serialized for user debugging
+  pub(crate) working_directory: PathBuf,
```

Comment:
This should probably be the file path, since you can have multiple justfiles in the same directory

Reply:
Yes, but the same justfile could be used in multiple directories :)

What I probably need to do is use the justfile path and working directory to be part of the hash that changes the cache filename.